### ISSUE TITLE
Changing the team should not refresh everything

### ISSUE THREAD

**Issue by Ukochka** [grafana/oncall#1492]
I  wish switching teams did not do a full page/ui refresh resulting in init/sync being called, it would be nice if it felt more like a filter and only updated the contents of the tables. @mderynck

**Comment by Ukochka** [comment]
We have a proposal from @raphael-batte  to remove this dropdown from top-right corner
![image (3)](https://user-images.githubusercontent.com/20494436/223701530-f44d0bc6-2371-489e-ab5a-dda702cbde33.png)

**Comment by greatvovan** [comment]
Shouldn't this one be closed? I see the teams selector was implemented as one of filters in the list.

**Comment by maskin25** [comment]
yes, now team selector is implemented as filter, so I'm closing this issue
